int16 flag
---
int16 node_x
int16 node_y 
